source/blog/tag.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_atom_feed_to_head %>
```

Comment:
`ubxd_web` did it so I just duplicated that.  It might be better to allow for tag based feeds and author based feeds, but it's a bit fancy.  Your call: we can remove it or leave it and decide later if we want to add more scoped feeds.